lemminx-maven/src/test/resources/pom-remote-artifact-download-hover.xml

```diff
@@ -0,0 +1,28 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.test</groupId>
+	<artifactId>test</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<packaging>pom</packaging>
+
+
+	<dependencies>
+	<dependency>
+		<groupId>org.glassfish.jersey</groupId>
+		<artifactId>project</artifactId>
+		<version>2.19</version>
+	</dependency>
+	</dependencies>
+
+	<repositories>
```

Comment:
Try removing the <repository> to see if this test still passes? ie. is lemminx-maven actually downloading the artifact from maven central

Reply:
@mickaelistria this confirms that the downloads are occurring from maven central and not other repositories listed in the pom.